engine.py

```diff
@@ -494,9 +494,11 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
         # newly created widget with the same Id.
         for widget in QtGui.QApplication.allWidgets():
             if widget.objectName() == panel_id:
+                widget.close()
                 widget.deleteLater()
+                self._tk_vred._dock_widgets.clear()
```

Comment:
if you clear the whole _dock_widgets dictionary, you can get rid of other panels that could be registered
in the default config, we only have SG Panel, but be careful that custom configs can registered apps as panel
so for me, it would be better to properly delete the widget item instead of clearing the entire dict